### DISCUSSION TITLE
error in Npm installation

### DISCUSSION THREAD

**Discussion by MadanLad** [community/community#46607] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

while npm install I am facing following error
npm ERR! Windows_NT 10.0.19044
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code EPROTO
npm ERR! errno EPROTO
npm ERR! syscall write

npm ERR! write EPROTO 101057795:error:1408D07B:SSL routines:ssl3_get_key_exchange:bad signature:openssl\ssl\s3_clnt.c:2032:
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Documents\New_folder\npm-debug.log

**Comment by jge162** [comment]
try clearing the npm cache by running 
`npm cache clean --force`
